lib/src/core/message/components/component_style.dart

```diff
@@ -1,6 +1,7 @@
 import 'package:nyxx/src/utils/enum.dart';
 
 /// Style for a button.
+@Deprecated("Use ButtonStyle")
```

Comment:
Why mark as deprecated if a major version change is happening? Just removing it would be cleaner.

Reply:
thats a mistake, i mean major version increment since these changes are not breaking if not necessary this PR will be released as minor version. Im still targeting next since no one knows when modals gonna release. Will fix that version constraint

Reply:
done